hack/create.sh

```diff
@@ -63,11 +63,14 @@ export RESOURCEGROUP=$1
 rm -rf _data
 mkdir -p _data/_out
 
-ttl=76h
-if [[ -n $RESOURCEGROUP_TTL ]]; then
-  ttl=$RESOURCEGROUP_TTL
+if [[ -z "$NOGROUPTAGS" ]]; then
+  ttl=76h
+  if [[ -n "$RESOURCEGROUP_TTL" ]]; then
+    ttl=$RESOURCEGROUP_TTL
+  fi
+  GROUPTAGS=--tags now=$(date +%s) ttl=$ttl
```

Comment:
This one broke our GC, fixing in https://github.com/openshift/openshift-azure/pull/601

Reply:
oops, thanks @kargakis